### ISSUE TITLE
Need an L1 barrier after L1 zero-ing at device init

### ISSUE THREAD

**Issue by davorchap** [tenstorrent/tt-metal#2690]
We need to ensure that L1 zero-ing fully finishes before we start any launch of fast dispatch FW or slow dispatch FW download.

**Comment by abhullar-tt** [comment]
> I'm keeping this open until we figure out:
> 
> * PCIe TLB regions and their mapping to WC / UC on the host

Can we close this issue since we aren't clearing L1 on init/close anymore and open another issue for PCIe TLB regions?

**Comment by pgkeller** [comment]
> > I'm keeping this open until we figure out:
> > 
> > * PCIe TLB regions and their mapping to WC / UC on the host
> 
> Can we close this issue since we aren't clearing L1 on init/close anymore and open another issue for PCIe TLB regions?

yes
